app/controllers/providers/correspondence_address/care_ofs_controller.rb

```diff
@@ -10,11 +10,20 @@ def show
       def update
         @form = Addresses::CareOfForm.new(form_params)
 
+        delete_care_of_information if form_params[:care_of].eql?("no")
```

Comment:
Does it follow our standard pattern to do this update in the form class?

Reply:
Updated in final commit